packages/api/README.md

```diff
@@ -33,7 +33,7 @@ npm start
 
 To test Stripe integration, you need to create a Stripe account and get the API keys.
 
-Then, you need to add in your `packages/api/.env` file in the following content:
+Then, you need to add your `packages/api/.env` file in the following content:
```

Comment:
Hi, thank you for you contribution. 

I think this should read:

`Then, you need to add in the following variables in your `packages/api/.env` file :`

Reply:
Yeah I'll make sure to change it.
Thanks @anfibiacreativa ✨